api/src/main/java/org/openmrs/module/eptsreports/metadata/HivMetadata.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
+ * compliance with the License. You may obtain a copy of the License at
+ * http://license.openmrs.org
+ *
+ * Software distributed under the License is distributed on an "AS IS"
+ * basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See the
+ * License for the specific language governing rights and limitations
+ * under the License.
+ *
+ * Copyright (C) OpenMRS, LLC.  All Rights Reserved.
+ */
+package org.openmrs.module.eptsreports.metadata;
+
+import org.openmrs.Concept;
+import org.openmrs.EncounterType;
+import org.openmrs.Program;
+import org.openmrs.ProgramWorkflowState;
+import org.springframework.stereotype.Component;
+
+@Component("hivMetadata")
+public class HivMetadata extends CommonMetadata {
+	
+	//ECOUNTER TYPES
+	public static final String S_TARV_ADULTO_SEGUIMENTO = "e278f956-1d5f-11e0-b929-000c29ad1d07";
```

Comment:
Is there a reason why these uuids are public and static? Do they get referenced anywhere outside of this file?

Reply:
Fixed

Reply:
Ok cool, I'll add a ticket to our backlog to add some tests when we have some

Reply:
yes as discussed, lets keep them private and limit the use to the 'get' methods for retrieving these values